src/Extensions/Microsoft.Diagnostics.Monitoring.Extension.Common/EgressProvider.cs

```diff
@@ -11,12 +10,10 @@ namespace Microsoft.Diagnostics.Monitoring.Extension.Common
 {
     internal abstract class EgressProvider<TOptions> where TOptions : class
     {
-        protected ILogger Logger { get; set; }
-
-        public EgressProvider() { }
+        protected EgressProvider() { }
 
         public abstract Task<string> EgressAsync(
-            ILogger logger,
+            string providerName,
```

Comment:
Is there a reason why we're passing this here? I couldn't find how S3/Azure were using it

Reply:
These two extensions don't need it but it's there for if we add other extensions that need to look up configuration external from dotnet-monitor. I suppose the argument would be that all of the extensions that we maintain should get their configuration from dotnet-monitor so this wouldn't be necessary. I can remove it for now.